tests_e2e/orchestrator/docker/Dockerfile

```diff
@@ -46,17 +45,8 @@ RUN \
     #                                                                                                            \
     # Install the Azure CLI                                                                                      \
     #                                                                                                            \
-    apt-get install ca-certificates curl apt-transport-https lsb-release gnupg                                && \
-    mkdir -p /etc/apt/keyrings                                                                                && \
-    curl -sLS https://packages.microsoft.com/keys/microsoft.asc                                                  \
-        | gpg --dearmor                                                                                          \
-        | tee /etc/apt/keyrings/microsoft.gpg > /dev/null                                                     && \
-    chmod go+r /etc/apt/keyrings/microsoft.gpg                                                                && \
-    AZ_REPO=$(lsb_release -cs)                                                                                && \
-    echo "deb [arch=`dpkg --print-architecture` signed-by=/etc/apt/keyrings/microsoft.gpg] https://packages.microsoft.com/repos/azure-cli/ $AZ_REPO main" \
-      | tee /etc/apt/sources.list.d/azure-cli.list                                                            && \
-    apt-get update                                                                                            && \
-    apt-get install azure-cli                                                                                 && \
+    tdnf -y install azure-cli                                                                                 && \
```

Comment:
azure-cli is available in mariner repository. The repo link https://cvedashboard.azurewebsites.net/#/packages